pybricks/parameters/pb_type_image.c

```diff
@@ -44,8 +44,10 @@ static int get_color(mp_obj_t obj) {
         return mp_obj_get_int(obj);
     }
     const pbio_color_hsv_t *hsv = pb_type_Color_get_hsv(obj);
-    int32_t v = pbio_int_math_bind(hsv->v, 0, 100);
-    return max - v * max / 100;
+    uint16_t h = pbio_int_math_bind(hsv->h, 0, 359);
+    uint8_t s = pbio_int_math_bind(hsv->s, 0, 100);
+    uint8_t v = pbio_int_math_bind(hsv->v, 0, 100);
```

Comment:
The range of these values should already be guaranteed. If not, we could have problems elsewhere as well. Maybe just an `assert()` here instead to catch problems in debug builds?

Reply:
I think https://github.com/pybricks/pybricks-micropython/pull/421 fixes that.